src/libs/import.c

```diff
@@ -736,7 +747,19 @@ static guint _import_set_file_list(const gchar *folder, const int folder_lgth,
       if(include_jpegs || (ext && g_ascii_strncasecmp(ext, ".jpg", sizeof(".jpg"))
                                && g_ascii_strncasecmp(ext, ".jpeg", sizeof(".jpeg"))))
       {
-        const gboolean already_imported = dt_images_already_imported(fullname);
```

Comment:
I think removing it altogether is the most clean solution.

Reply:
re: avoid the filmid query in the case of a directory with only subdirs. No, I don't think that's worth the effort. Too much complexity for a minimal gain.

Reply:
I really don't agree here. The function as a simple and clean semantic and could be used later. Please keep it to ensure that people will be reusing it instead of creating more copy in other part of dt. I just did that in my current GUI rework PR. A part of the code I needed was in a .c and not exported. Hopefully I have spotted it otherwise I would have written again the same code.